src/drivers/imx/sai.c

```diff
@@ -0,0 +1,311 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2019 NXP
+//
+// Author: Daniel Baluta <daniel.baluta@nxp.com>
+// Author: Jerome Laclavere <jerome.laclavere@nxp.com>
+// Author: Guido Roncarolo <guido.roncarolo@nxp.com>
+
+#include <errno.h>
+#include <stdint.h>
+#include <sof/audio/component.h>
+#include <sof/bit.h>
+#include <sof/drivers/edma.h>
+#include <sof/drivers/sai.h>
+#include <sof/lib/dai.h>
+#include <sof/lib/dma.h>
+#include <ipc/dai.h>
```

Comment:
From most local to global in alphabetical order:
```
#include <sof/audio/component.h>
#include <sof/bit.h>
#include <sof/drivers/edma.h>
#include <sof/drivers/sai.h>
#include <sof/lib/dai.h>
#include <sof/lib/dma.h>
#include <ipc/dai.h>
#include <errno.h>
#include <stdint.h>
```

Reply:
thanks! 
:)